### ISSUE TITLE
Call doesn't end after sending voicemail

### ISSUE THREAD

**Issue by SomeKoder** [VoiSmart/pjsip-android#140]
Steps to reproduce:
- Make a call on PhoneA using pjsip
- Decline the incoming call on PhoneB
- Record voicemail
- Send "*" with DTMF when assistance says "Press star to send voicemail and end call"
- Assistant says "Goodbye"
- Call status remains the same, no CallStatusEvent updates, nothing comes through onCallStats either

Our iOS app seems to receiving the BYE packet juste fine so I think maybe this case is not implemented in this library.

**Comment by aenonGit** [comment]
does the server sends the BYE? If so, does your app receives it?

**Comment by SomeKoder** [comment]
Hey I'm very sorry about the delay! Got caught up with other tasks. I'm going to close this issue for now due to not having enough information! Again sorry for the delay

**Comment by SomeKoder** [comment]
`accountConfig.getNatConfig().setContactRewriteMethod(4);`

Removing this config fixed the issue, I'm guessing the port kept changing so the backend didn't know where to send the BYE packet anymore 😃

**Comment by SomeKoder** [comment]
So turns out it did not fix the issue. I did find out how to reproduce though. 

Turns out, if a call is less than 15 seconds long, and the other person hangs up, I receive the BYE packet no problem.

If the call is more than 45 seconds long, and the other person hangs up, I don't receive the BYE packet at all, so the call shows as still in progress. Wireshark doesn't catch the packet either. I checked with the backend and it's definitely sending the packet, I'm just not receiving it on my end. 

Any ideas?
